src/main/java/uk/gov/hmcts/darts/arm/client/ArmApiClient.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.darts.arm.client;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import uk.gov.hmcts.darts.arm.client.model.UpdateMetadataRequest;
+import uk.gov.hmcts.darts.arm.client.model.UpdateMetadataResponse;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@FeignClient(
+    name = "arm-api-client",
+    url = "${darts.storage.arm-api.url}"
+)
+public interface ArmApiClient {
```

Comment:
Consider writing a dedicated integration test to cover the client functionality, using embedded Wiremock to verify the request similar to `ArmTokenClientIntTest`. The added value is to ensure the client is making the request in the expected way, in this case a `POST` with a bearer token in the header, and a JSON request body.

When I was writing the token client I found this test useful because Feign client was silently making unwanted changes to the request, which prompted some necessary workarounds.

Reply:
Yep, I spotted you were doing something with wiremock in darts-api.
I was initially wanting to check against our darts-stub-services on staging to check I got the OK response ok actually, but I found that it was giving me Error 500 more often than not yesterday, so the deployment there doesn't feel too stable for some reason. Therefore I just ended up mocking the clients and checking the parameters.

Reply:
Done in commit 4bb0083d82cade406a78ca36c0948fb6032cd80f